README.md

```diff
@@ -1 +1,92 @@
-# Covid-19-app
\ No newline at end of file
+# CovidTracker--SARS-CoV-2.
+
+> This webapp provides Covid-19 Tracking information using the data it gets from an [API data source](https://corona.lmao.ninja) to provide comprehensive information about the novel coronavirus,
+>
+> It is created with [Create-React-App](https://github.com/facebook/create-react-app)
+
+---
+## Video Presentation
+https://www.loom.com/share/eeaba57761c74b7da3480e4d52010a4b
```

Comment:
- [ ] Please, try fixing this issue mentioned by the previous reviewer: your video presentation is recorded by another user and with old dat
<img width="1201" alt="Screen Shot 2022-07-17 at 00 35 22" src="https://user-images.githubusercontent.com/49064106/179371215-d5a0b05c-c1c5-4429-a676-1d91602163b7.png">
e